pkg/util/util.go

```diff
@@ -169,24 +169,20 @@ func EmbeddedClusterVersion() string {
 func ReplicatedAPIEndpoint(license *kotsv1beta1.License) (string, error) {
 	if IsEmbeddedCluster() {
 		if endpoint := os.Getenv("REPLICATED_API_ENDPOINT"); endpoint != "" {
-			return maybePrependHTTPS(endpoint), nil
+			return endpoint, nil
 		}
 		return "", errors.New("REPLICATED_API_ENDPOINT environment variable is required")
 	}
 
 	if license != nil && license.Spec.Endpoint != "" {
-		return maybePrependHTTPS(license.Spec.Endpoint), nil
+		if strings.HasPrefix(license.Spec.Endpoint, "http://") || strings.HasPrefix(license.Spec.Endpoint, "https://") {
+			return license.Spec.Endpoint, nil
+		}
+		return fmt.Sprintf("https://%s", license.Spec.Endpoint), nil
```

Comment:
I don't think you need to do this - our APIserver code includes the scheme in the license format (and it's not user/vendor editable)

Reply:
removed